### DISCUSSION TITLE
[PvP] Custom Reductions WTF?

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#6995] [General]
So,

we had people ask
- why AV has had different tunings compared to other BGs
- why healing / absorbs where reduced so much

we got:
- https://discord.com/channels/788063059926712341/794247581763895316/1226276548844453938
After a thorough review of the current situation, **all tuning has been removed from Battlegrounds**, while the minimum tuning that was in place before will remain in Arenas and the Boatwrighte's duel-area for now.
We will keep monitoring the situation to ensure a pleasant PvP experience for our players.

I am just wondering if you took into account feedback from players that play classes other than 
- warrior
- rogue
- firemage

in battlegrounds?

The bigger question is: **what are you trying to achieve?**
do you want to offer a 
- tbc like pvp experience
- a wotlk like pvp experience
- a minimum tolerable pvp experience
- a wotlk-prepatch like experience

so far, all the tuning, different tunings in arena, bgs (also different tuning between bgs), seems rather arbitrary from the outside. What do you strive to provide for s2-s4 until we get wotlk?
(wotlk in itself will technically be an issue again because 3.3.5 talents and abilities are designed around s8, not s5 - so the same idea or concept should keep applying there)

**Comment by Amrx2** [comment]
@balleny redo this in a respectful manner without all the insults to the team and players who fed back into this please. Closing.
